flake.nix

```diff
@@ -7,14 +7,6 @@
     url = "github:onekey-sec/unblob-native";
     inputs.nixpkgs.follows = "nixpkgs";
   };
```

Comment:
do we still need to declare unblob-native here ?

Reply:
since unblob-native is part of https://github.com/NixOS/nixpkgs/pull/345554 and pulled by unblob, I would think we can drop it in a similar fashion than `lzallright` and `pyperscan`

Reply:
Not really, but I kept in as these two components are tied together very closely. Will work on #1034 next anyway.

Reply:
ok